patch-package
new file mode 100644
Binary files /dev/null and b/node_modules/subscribe-ui-event/.index.es.js.swp differ
--- a/node_modules/subscribe-ui-event/index.es.js
+++ b/node_modules/subscribe-ui-event/index.es.js
@@ -7,7 +7,7 @@ import listenLib from './dist-es/lib/listen';
 import subscribeLib from './dist-es/subscribe';
 import unsubscribeLib from './dist-es/unsubscribe';
 
-const IS_CLIENT = typeof window !== 'undefined';
+var IS_CLIENT = typeof window !== 'undefined';
 
 function warn() {
   if (process.env.NODE_ENV !== 'production') {
@@ -15,6 +15,6 @@ function warn() {
   }
 }
 
-export const listen = IS_CLIENT ? listenLib : warn;
-export const subscribe = IS_CLIENT ? subscribeLib : warn;
-export const unsubscribe = IS_CLIENT ? unsubscribeLib : warn;
+export var listen = IS_CLIENT ? listenLib : warn;
+export var subscribe = IS_CLIENT ? subscribeLib : warn;
+export var unsubscribe = IS_CLIENT ? unsubscribeLib : warn;
